<launch>

  <node pkg="annotated_planar_patch_map" type="annotation_pcd_lifter_node" 
     	   name="annotation_lifter" respawn="false" output="screen" 
	>

<!--	launch-prefix="xterm -e gdb -args" -->


	   <param name="fixed_frame" value="/stereo_optical_frame"/>
	   <param name="local_fixed_frame" value="/odom_combined"/>

	   <param name="dist_tolerance" value="0.0"/>

	   <param name="lifting_delay" value="1.1"/>

	   <param name="interval_before" value="1.0"/>
	   <param name="interval_after" value="1.0"/>

	   
	   <param name="dist_tolerance" value="0.0"/>
	   <param name="max_depth" value="20.0"/>
	   <param name="min_depth" value="0.1"/>


	   <param name="use_color" value="True" type="bool"/>
	   <param name="annotate_reprojection" value="True" type="bool"/>
	   <param name="annotate_image_id" value="True" type="bool"/>

	   <param name="downsample" value="0"/>

	   <remap from="build-cloud" to="/laser_scan_assembler/build_cloud"/>	 
	   <remap from="cam_info" to="/stereo/left/cam_info"/>	 
<!--	   <remap from="annotations_2d" to="annotations_2d_repub"/>	  -->

	   <param name="cam_info_hist/max_msg" value="1200" />

	   <param name="image_hist/max_msg" value="30" />
	   <remap from="image" to="/stereo/left/image_rect"/>	 

     </node>	   

</launch>
